### PR TITLE
chore: remove serve others orders docs toggle (FPLA-3227)

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-3327

### Change description ###
Remove toggle used for serving others orders and docs.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java [hunk 1]
```diff
@@ -108,30 +103,25 @@ public void notifyAdmin(final AdditionalApplicationsUploadedEvent event) {
 
             NotifyData notifyData = contentProvider.getNotifyData(caseData);
             String recipient = courtService.getCourtEmail(caseData);
-            String template = featureToggleService.isServeOrdersAndDocsToOthersEnabled()
-                              ? UPDATED_INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC
-                              : INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE;
-
-            notificationService.sendEmail(template, recipient, notifyData, caseData.getId());
+            notificationService.sendEmail(
+                UPDATED_INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC, recipient, notifyData, caseData.getId());
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java:107]: would it be worth updating this template name now that the old one is removed?

**Review by lewisbirks — DISMISSED** [review]

File: service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java (hunk 1 above)
Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java:107]: it seems that there is another that is defined as `INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_CTSC` any reason for that?

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java:107]: looks like it is identical to the one that you removed, will remove that one as well and update the name of the current one

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java [hunk 2]
```diff
@@ -83,14 +81,12 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
 
         caseDetails.getData().putAll(applicationsFeeCalculator.calculateFee(caseData));
 
-        if (featureToggleService.isServeOrdersAndDocsToOthersEnabled() && caseData.hasRespondentsOrOthers()) {
-            caseDetails.getData().put("hasRespondentsOrOthers", "Yes");
-            caseDetails.getData().put("people_label", peopleInCaseService.buildPeopleInCaseLabel(
-                caseData.getAllRespondents(), caseData.getOthers()));
+        caseDetails.getData().put("hasRespondentsOrOthers", "Yes");
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java:84]: why has the `caseData.hasRespondentsOrOthers` been removed?

Comment (aliveni) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java:84]: I'll put it back, I think it was removed while resolving the merge conflicts

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**